django/contrib/postgres/functions.py

```diff
@@ -0,0 +1,16 @@
+"""
+Classes that represent PostgreSQL database functions.
+"""
+from django.db.models.expressions import Func, Value
+
+
+class Similarity(Func):
+    """
+    Trigram similarity function.
```

Comment:
I don't think that this docstring is useful, it is describing what we are actually can read from the code, and the code is obvious.